internal/verification/config.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package verification
+
+import "time"
+
+// Config represents the available configuration for the public health authority
+// verification piece.
+type Config struct {
+	CacheDuration time.Duration `env:"HEALTH_AUTHORITY_CACHE_DURATION, default=5m"`
```

Comment:
Should this be `VERIFICATION_CACHE_DURATION`?

Reply:
done